.gitignore

```diff
@@ -11,3 +11,4 @@ config/samples
 dist
 bin
 manager_image_patch.yaml-e
+cover.out
```

Comment:
this change seems out of scope for this PR.

Reply:
I can backout that change, but `make test` will produce this https://github.com/kubernetes-sigs/cluster-api-provider-docker/blob/master/Makefile#L33

Reply:
I'm ok with this change going through. @neolit123 do you think this is worth blocking on?